ecs-init/docker/docker.go

```diff
@@ -427,6 +432,65 @@ func getDockerPluginDirBinds() []string {
 	return pluginBinds
 }
 
+// take pathPredicate as an argument for unit testing
+func getCapabilityExecBinds(pathPredicate func(path string, predicate func(fileInfo os.FileInfo) bool) (bool, error)) []string {
+	const (
+		capabilityExecName = "exec"
+
+		hostCapabilitiesResourcesRootDir      = "/home/ec2-user/capabilities-deps"
+		containerCapabilitiesResourcesRootDir = "/capabilities"
+
+		hostBinRelativePath = "bin"
+		hostCertsDir        = "/etc/pki/ca-trust/extracted/pem"
+		hostCertsFilename   = "tls-ca-bundle.pem"
+
+		containerBinRelativePath   = "bin"
+		containerCertsRelativePath = "certs"
+
+		// SSM-related settings are not configurable for now
+		// containerCapabilityExecConfigPath = "config"
+	)
+
+	isDir := func(fileInfo os.FileInfo) bool {
+		return fileInfo.IsDir()
+	}
+	isFile := func(fileInfo os.FileInfo) bool {
+		return !fileInfo.IsDir()
+	}
+	hostCapabilityExecResourcesDir := filepath.Join(hostCapabilitiesResourcesRootDir, capabilityExecName)
+	containerCapabilityExecResourcesDir := filepath.Join(containerCapabilitiesResourcesRootDir, capabilityExecName)
+
+	binds := []string{}
+
+	// bind mount the entire /deps/exec/bin folder for higher flexibility
```

Comment:
the path mentioned here and `hostCapabilitiesResourcesRootDir` are different. Could you update the comment?

Reply:
this comment is intended to use an abstracted path since we might later make this configurable; yet good point, I'll update the path to `/host/path/dependency/exec/bin` so it's more obvious in terms of what it means